produce.go

```diff
@@ -10,6 +10,7 @@ import (
 	"time"
 
 	"github.com/Shopify/sarama"
+	"github.com/burdiyan/kafkautil"
```

Comment:
🌷  This is introducing several transitive dependencies that are only used by part of kafkautil that we don't use. An alternative could be to just copy https://github.com/burdiyan/kafkautil/blob/master/partitioner.go and its test to our repo, following "A little copying is better than a little dependency." from the [proverbs](https://go-proverbs.github.io/). The implementation is unlikely to be changed given the last change was done more than 2 years ago.

Reply:
I agree on the philosophy but I'd have done that in `heetch/felice` instead not here as this is a CLI utility, the impact is lower. Best option would have to have a single package that both `felice` and `hkt` imports from.

Reply:
Yes the impact is low here, let's keep the import 👌